tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -18,8 +18,19 @@ Suite Setup         RHOSi Setup
 Resource            ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
 Resource            ../../../Resources/Common.robot
 
-
 *** Test Cases ***
+Verify Dashbord has no message with NO Component Found
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1493
+    [Documentation]   Verify the service appears in Applications > Enabled
+    [Setup]   Test Setup For Rhods Dashboard
+    Oc Apply  kind=Subscription  src=tests/Tests/100__deploy/100__installation/bad_subscription.yaml
+    Menu.Navigate To Page    Applications    Enabled
```

Comment:
I think you need to reload the page otherwise latest config will not get reflected as soon as changes are mode.
**Reload Page** is the keyword

Reply:
Also i think you need to restart the rhods dashboard pods